src/main/java/co/org/osso/wallet/AccountService.java

```diff
@@ -0,0 +1,26 @@
+package co.org.osso.wallet;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class AccountService {
+
+    ListAccountsResponse listAccounts(){
+        ListAccountsResponse accountList = new ListAccountsResponse();
+        for (int i = 0; i < 10; i++){
+            Account account = new Account();
+            account.setId(Long.valueOf(i));
+            account.setAccount("user_" + i);
+            account.setFullName("Test User " +i);
```

Comment:
Let's run the formatter on all our code. I think this would look different afterwards.